composer.json

```diff
@@ -24,11 +24,12 @@
         "doctrine/annotations": "^1.8",
         "doctrine/cache": "^1.9.1",
         "doctrine/collections": "^1.5",
-        "doctrine/common": "^2.11",
+        "doctrine/common": "^2.11 || ^3.0",
         "doctrine/dbal": "^2.9.3",
         "doctrine/event-manager": "^1.1",
+        "doctrine/inflector": "^1.0",
```

Comment:
Using the same constraint as in `doctrine/common` 2 here

Reply:
why, is it needed?